src/components/Button/Button.js

```diff
@@ -18,16 +18,23 @@ const { defaultProps, propTypes } = CarbonButton;
 
 // TODO: V3 - Remove deprecated props `largeText`.
 
-const Button = ({ className, largeText, size, ...other }) => {
+const Button = ({ className, largeText, size, kind, ...other }) => {
   const isSize = value => size === value;
   const isLarge = isSize('large') || largeText || isSize('lg') || isSize('xlg');
 
+  const isKind = value => kind === value;
```

Comment:
- [ ] Thanks! The final thing here is to remove this line now it's no longer needed:

```suggestion
```